docs/arcaflow/workflows/expressions.md

```diff
@@ -261,6 +261,7 @@ Functions:
 | `toUpper(string)`                                | string       | Returns the input with Unicode letters mapped to their upper case.                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `splitString(string, string)`                    | list[string] | Returns a list of the substrings which appear between instances of the specified separator; the separator instances are not included in the resulting list elements; adjacent occurrences of separator instances as well as instances appearing at the beginning or ending of the input will produce empty string list elements.<br>&nbsp; Param 1: The string to split.<br>&nbsp; Param 2: The separator.                                                                                     |
 | `readFile(string)`                               | string |  Returns the contents of a file as a UTF-8 character string, given a file path string. Relative file paths are resolved from the Arcaflow process working directory. Shell environment variables are not expanded.                                                                                     |
+| `bindConstants(list[any], any)`                               | list[object] |  Returns a list of objects with a new ID equal to schema name of the first parameter concatenated with the schema name of the second parameter. For each item in the list, an object with two properties is created. The item is assigned to this object's `item` property. The second argument of the `bindConstants` function is assigned to this object's `constant` property, thus repeating the second argument for each item in the list.                                                                                   |
```

Comment:
Given that there is a tradeoff between precision and clarity, I opted in my earlier suggestion to gloss over the separator.  (That is, just because we say "concatenated with" doesn't mean we _don't_ stick other stuff in there....)

I would pick at other nits, here.  Like, I don't think we need to stipulate that the list returned has a _new_ ID -- everything has an ID, and this list has one, too (so, we can just omit "new").  I'm not crazy about "equal" here, since we're not really comparing quantities -- rather we're describing what will be returned (I would use something like "composed", "consisting of", "formulated", or just "which is").

Anyway, I would again plug [my suggested text](https://github.com/arcalot/arcalot.github.io/pull/81#discussion_r1589854819).

Reply:
maybe i did this?